modules/examples/personas/stack.py

```diff
@@ -0,0 +1,47 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from typing import Any, Optional
+
+import cdk_nag
+from aws_cdk import Environment, Stack
+from constructs import Construct
+
+from personas import Personas as PersonasConstruct
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+class Personas(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        bucket_name: Optional[str],
+        env: Environment,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(scope, construct_id)
+
+        s3_bucket_prefix = bucket_name or f"{construct_id}-bucket"
```

Comment:
Can you confirm that this (`f"{construct_id}-bucket"`) is what we want? The readme doesn't mention it anywhere.

I think we should either make the bucket mandatory, or just not include permissions with a made-up bucket name.